source/daplink/drag-n-drop/vfs_manager.c

```diff
@@ -615,12 +615,15 @@ static void transfer_update_file_info(vfs_file_t file, uint32_t start_sector, ui
 static void transfer_reset_file_info()
 {
     vfs_mngr_printf("vfs_manager transfer_reset_file_info()\r\n");
-    if (file_transfer_state.stream_open) {
-        transfer_update_state(ERROR_ERROR_DURING_TRANSFER);
-    } else {
-        file_transfer_state = default_transfer_state;
-        abort_remount();
-    }
+    //check if the data started streaming; size can be updated on matching start sector and stream type
+    if(file_transfer_state.stream_started && (TRASNFER_FINISHED != file_transfer_state.transfer_state)){
```

Comment:
You should only need to check `stream_started` here, as it only gets reset back to 0 on remount.